README.md

```diff
@@ -95,68 +66,40 @@ The following components are provided by the customers:
 other database configurations are in the roles/tpa_single_node/vars/main.yml
 
-Postgres ssl mode is enabled by default. To disable it please change the following in vars/main.yml file:
+Postgres ssl mode is enabled by default. To disable it please change the following in roles/tpa_single_node/vars/main.yml file:
 `tpa_single_node_pg_ssl_mode: disable`.
 
+* [Trustification-PostgreSQL](https://github.com/trustification/trustification/blob/release/1.2.z/docs/modules/admin/pages/cluster-preparing.adoc#rds)
 
-### S3 or S3 compatible service like Minio
+
+### S3 or S3 compatible service
   Have the following unversioned S3 bucket names created:
   ```
   bombastic-default
   vexination-default
   v11y-default 
   ```
-  configured in the main.yml
-
-
-*  Details about how to configure the services can be found here [RHTPA external services deploy](https://docs.redhat.com/en/documentation/red_hat_trusted_profile_analyzer/1/html-single/deployment_guide/index#installing-trusted-profile-analyzer-by-using-helm-with-other-services_deploy)
-* [Trustification](https://github.com/trustification/trustification/blob/main/docs/modules/admin/pages/cluster-preparing.adoc)
-
+  configured in the roles/tpa_single_node/vars/main.yml
 
+* [Trustification S3](https://github.com/trustification/trustification/blob/release/1.2.z/docs/modules/admin/pages/cluster-preparing.adoc#s3-storage)
 
 
 Utilize the steps below to understand how to setup and execute the provisioning.
 
-## Installation
-
-
-Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
-
-```
-ansible-galaxy collection install redhat.trusted_profile_analyzer
-```
-
-You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
-
-
-```yaml
-collections:
-  - name: redhat.trusted_profile_analyzer
-```
-
-Note that if you install any collections from Ansible Galaxy, they will not be upgraded automatically when you upgrade the Ansible package.
-To upgrade the collection to the latest available version, run the following command:
-
-```
-ansible-galaxy collection install redhat.trusted_profile_analyzer --upgrade
-```
-
-You can also install a specific version of the collection, for example, if you need to downgrade when something is broken in the latest version (please report an issue in this repository). Use the following syntax to install version 0.2.0:
+## Configurations on the controller node
 
-```
-ansible-galaxy collection install redhat.trusted_profile_analyzer:==0.2.0
-```
-
-## Verifying the deployment
+On the controller node  export the following env vars
 
-1. Export the following environment variables, replacing `TODO` with your relevant information:
+1. Export the following environment variables, replacing the placeholders with your relevant information:
 
    ```shell
       export TPA_SINGLE_NODE_REGISTRATION_USERNAME=<Your Red Hat subscription username>
```

Comment:
I hope we could remove TPA_SINGLE_NODE_REGISTRATION_USERNAME and TPA_SINGLE_NODE_REGISTRATION_PASSWORD can be removed as they are used in vagrant configuration.

Reply:
applied